### DISCUSSION TITLE
API 정보 저장 관련 질문

### DISCUSSION THREAD

**Discussion by tjrkdgnl** [solvedac/unofficial-documentation#41] [질답] [answered]
안녕하세요 관리자님. solvedac API를 이용해서 앱을 만들고 있습니다. 앱 개발 중에 API 호출이 너무 잦으면 blocking된다는 것을 알게되었습니다. 따라서 API 호출로부터 response 값인 TaggedProblem을 firebase에 저장해서 관리하려고 하는데 저작권 문제 등에 걸리는 일일까요?

**Comment by RanolP** [comment, marked as answer]
이에 관해서는 shiftpsh 님께 여쭤봐야 정확히 알 수 있겠습니다만 큰 문제는 없을 거라 생각합니다.
